services/Service.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const apiClient = axios.create({
+  baseURL: `http://localhost:3000`,
+  withCredentials: false, // This is the default
+  headers: {
+    Accept: "application/json",
+    "Content-Type": "application/json"
+  }
+});
```

Comment:
Fajnie, że używasz axiosa. Jest to najbardziej w tym momencie popularny klient http. Z tym, że nigdzie go nie importujesz :P. Więc w zasadzie go nie używasz.